dist/ovh-angular-otrs.js

```diff
@@ -537,6 +536,10 @@ angular.module("ovh-angular-otrs")
                 $scope.$on("otrs.popup.toggle", $scope.toggle);
                 $scope.$on("otrs.popup.open", $scope.open);
                 $scope.$on("otrs.popup.close", $scope.close);
+
+                $element.children(".otrs-popup.draggable").on("drag", function (event, ui) {
```

Comment:
remove listener or we should use `one` instead of `on`

Reply:
this is done.